common/flavorization/00_CTP.txt

```diff
@@ -6526,6 +6526,399 @@ duke_tribal_female_italian = {
 ## End of Latin_cultrue_group ##
 ###############################################
 
+
+###############################################
+## Eastern_slavic_cultrue_group ##
+###############################################
+
+baron_male_east_slavic_group = {
+	type = character
+	gender = male
+	special = holder
+	tier = barony
+	priority = 7
+	cultures = { east_slavic_group }
```

Comment:
I do not know if this works. 
In the vanilla, culture groups are defined using the condition
`culture_groups = { foo bar }`

Reply:
If this works, it's ready.

Reply:
> 
> 
> I do not know if this works.
> In the vanilla, culture groups are defined using the condition
> `culture_groups = { foo bar }`

Oh, I didn't know that. Fixed.